boards/arm/nrf54l15pdk_nrf54l15/nrf54l15pdk_nrf54l15_cpuapp.dts

```diff
@@ -133,13 +133,30 @@
 		compatible = "fixed-partitions";
 		#address-cells = <1>;
 		#size-cells = <1>;
-		slot0_partition: partition@0 {
-			label = "image-0";
+		boot_partition: partition@0 {
+			label = "mcuboot";
 			reg = <0x0 DT_SIZE_K(64)>;
 		};
-		storage_partition: partition@f2000 {
+		slot0_partition: partition@10000 {
+			label = "image-0";
+			reg = <0x10000 DT_SIZE_K(349)>;
+		};
+		slot0_ns_partition: partition@67400 {
+			label = "image-0-nonsecure";
+			reg = <0x67400 DT_SIZE_K(349)>;
+		};
+		slot1_partition: partition@be800 {
+			label = "image-1";
+			reg = <0xbe800 DT_SIZE_K(349)>;
+		};
+		slot1_ns_partition: partition@115c00 {
+			label = "image-1-nonsecure";
+			reg = <0x115c00 DT_SIZE_K(349)>;
```

Comment:
Hate to be a pain but can we make this divisable by 4 so that external memory devices e.g. mx25 can be used, otherwise it is impossible? 348KiB for each of these then add 4KiB to settings to make that 36KiB